pipeline/packer/jenkins-master.json

```diff
@@ -0,0 +1,51 @@
+{
+  "_description": "This template produces a Jenkins server.",
+  "variables": {
+    "repository": "{{env `DOCKER_REPOSITORY`}}",
+    "tag": "{{env `BUILD_TAG`}}",
+    "docker_registry_url": "{{env `DOCKER_REGISTRY_URL`}}"
+  },
+  "builders": [
+    {
+      "type": "docker",
+      "image": "jenkins/jenkins:lts",
+      "commit": true,
+      "changes": [
+        "ENV CASC_JENKINS_CONFIG /usr/share/jenkins/ref/jenkins.yaml",
+        "ENTRYPOINT [\"/sbin/tini\", \"--\", \"/usr/local/bin/jenkins.sh\"]"
```

Comment:
For some reason, I was finding that packer wasn't maintaining the entrypoint. So I ended up copy-pasting it here from the base image.